WordPress/src/main/java/org/wordpress/android/ui/mysite/MySiteCardAndItem.kt

```diff
@@ -275,6 +276,33 @@ sealed class MySiteCardAndItem(open val type: Type, open val activeQuickStartIte
                     )
                 }
 
+                sealed class ActivityCard(
+                    override val dashboardCardType: DashboardCardType,
+                    open val footerLink: FooterLink? = null
+                ) : DashboardCard(dashboardCardType) {
+                    data class ActivityCardWithItems(
+                        val title: UiString,
+                        val activityItems: List<ActivityItem>,
+                        override val footerLink: FooterLink
+                    ) : ActivityCard(
+                        dashboardCardType = DashboardCardType.ACTIVITY_CARD,
+                        footerLink = footerLink
+                    ) {
+                        data class ActivityItem(
+                            val title: UiString,
+                            val subtitle: UiString?,
+                            @DrawableRes val primaryImage: Int,
+                            val iconColor: Int? = null,
+                            val onClick: ListItemInteraction
```

Comment:
We haven't set `primaryImage`, `iconColor` and `iconColor` from view holders. Will we use them in following PRs?

Reply:
Nope, they are gone. Just placeholder logic here.